Cargo.toml

```diff
@@ -10,3 +10,9 @@ edition = "2018"
 regex = "1"
 rhai = "0.7"
 ip_network = "0.3.4"
+diesel = { version = "1.4.2", features = ["r2d2", "postgres"], optional = true }
+
+[features]
+default = ["postgres"]
```

Comment:
How about we put this adapter to another workspace, so we can publish as 2 separate package to crate.io

Reply:
yes that will be a good idea, maybe later we can seperate adapters into workspaces